services/ymax-planner/src/engine.ts

```diff
@@ -383,6 +383,10 @@ export const startEngine = async (
         return BigInt((respData.header as any).height);
       case 'tendermint/event/Tx':
         return BigInt((respData.TxResult as any).height);
+      case 'tendermint/event/NewBlock': {
+        // @ts-expect-error
+        return BigInt(respData.block.header.height);
```

Comment:
The other cases are casting to `any` rather than expecting an error. 
```suggestion
        return BigInt((respData.block as any).header.height);
```